src/Magento/FunctionalTestingFramework/Util/Logger/MftfLogger.php

```diff
@@ -86,13 +81,10 @@ public function criticalFailure($message, array $context = [], $verbose = false)
     public function notification($message, array $context = [], $verbose = false)
     {
         $message = "NOTICE: " . $message;
-        // print during test generation
-        if ($this->phase === MftfApplicationConfig::GENERATION_PHASE && $verbose) {
+        // Suppress print during unit testing
+        if ($this->phase !== MftfApplicationConfig::UNIT_TEST_PHASE && $verbose) {
             print ($message . json_encode($context) . "\n");
         }
-        // suppress logging during test execution
-        if ($this->phase !== MftfApplicationConfig::EXECUTION_PHASE) {
-            parent::notice($message, $context);
-        }
+        parent::notice($message, $context);
```

Comment:
Can we safely revert the code recently done for trimming the log file size?

Reply:
reverted changes